### ISSUE TITLE
Slow image detection updates

### ISSUE THREAD

**Issue by oOMoeOo** [react-native-ar/react-native-arkit#185]
Hi,

i'm currently playing around with the image detection feature. For the moment i display a simple box on top of my detected image marker and the detection process works pretty well.
I'm using onAnchorDetected/onAnchorUpdated to reposition the shape using a simple position state.

My problem is, that there is a delay between moving the physical marker and updating the virtual object. When moving the marker a few centimeters, it takes more than one second to update the box position.

I've tested it in debug mode with different lighting environments using an iPhone 7 device. Am i doing something wrong or is it still an issue with the native ARKit or with RNARKit? I tested a similar approach with Unity/Vuforia and the marker tracking was awesome.

Kind regards

**Comment by macrozone** [comment]
that is a limitation of arkit, but i don't know how vuforia does overcome this.

however, upcoming ios 12 seems to allow for continues updates, see this page: https://developer.apple.com/documentation/arkit/arworldtrackingconfiguration/2941063-detectionimages?language=objc

**Comment by oOMoeOo** [comment]
@macrozone Yes, that's right. You can also see the effects in the following video (~minute 40)
https://developer.apple.com/videos/play/wwdc2018/610/
